chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -577,95 +652,6 @@ ifdef::VK_EXT_display_control[]
-
-endif::VK_VERSION_1_1[]
-ifdef::VK_KHR_swapchain_mutable_format[]
-ename:VK_IMAGE_CREATE_MUTABLE_FORMAT_BIT and
-ename:VK_IMAGE_CREATE_EXTENDED_USAGE_BIT_KHR are both set if
-ename:VK_SWAPCHAIN_CREATE_MUTABLE_FORMAT_BIT_KHR is set
-
-endif::VK_KHR_swapchain_mutable_format[]
-all other bits are unset
-endif::VK_VERSION_1_1,VK_KHR_device_group,VK_KHR_swapchain_mutable_format[]
-| pname:imageType               | ename:VK_IMAGE_TYPE_2D
-| pname:format                  | pname:pCreateInfo->imageFormat
-| pname:extent                  | {pname:pCreateInfo->imageExtent.width, pname:pCreateInfo->imageExtent.height, `1`}
-| pname:mipLevels               | 1
-| pname:arrayLayers             | pname:pCreateInfo->imageArrayLayers
-| pname:samples                 | ename:VK_SAMPLE_COUNT_1_BIT
-| pname:tiling                  | ename:VK_IMAGE_TILING_OPTIMAL
-| pname:usage                   | pname:pCreateInfo->imageUsage
-| pname:sharingMode             | pname:pCreateInfo->imageSharingMode
-| pname:queueFamilyIndexCount   | pname:pCreateInfo->queueFamilyIndexCount
-| pname:pQueueFamilyIndices     | pname:pCreateInfo->pQueueFamilyIndices
-| pname:initialLayout           | ename:VK_IMAGE_LAYOUT_UNDEFINED
-|====
-
-The pname:surface must: not be destroyed until after the swapchain is
-destroyed.
-
-If pname:oldSwapchain is dlink:VK_NULL_HANDLE, and the native window
-referred to by pname:surface is already associated with a Vulkan swapchain,
-ename:VK_ERROR_NATIVE_WINDOW_IN_USE_KHR must: be returned.
-
-If the native window referred to by pname:surface is already associated with
-a non-Vulkan graphics API surface, ename:VK_ERROR_NATIVE_WINDOW_IN_USE_KHR
-must: be returned.
-
-The native window referred to by pname:surface must: not become associated
-with a non-Vulkan graphics API surface before all associated Vulkan
-swapchains have been destroyed.
-
-Like core functions, several WSI functions, including
-fname:vkCreateSwapchainKHR return ename:VK_ERROR_DEVICE_LOST if the logical
-device was lost.
-See <<devsandqueues-lost-device,Lost Device>>.
-As with most core objects, sname:VkSwapchainKHR is a child of the device and
-is affected by the lost state; it must: be destroyed before destroying the
-sname:VkDevice.
-However, sname:VkSurfaceKHR is not a child of any sname:VkDevice and is not
-otherwise affected by the lost device.
-After successfully recreating a sname:VkDevice, the same sname:VkSurfaceKHR
-can: be used to create a new sname:VkSwapchainKHR, provided the previous one
-was destroyed.
-
-[NOTE]
-.Note
-====
-As mentioned in <<devsandqueues-lost-device,Lost Device>>, after a lost
-device event, the sname:VkPhysicalDevice may: also be lost.
-If other sname:VkPhysicalDevice are available, they can: be used together
-with the same sname:VkSurfaceKHR to create the new sname:VkSwapchainKHR,
-however the application must: query the surface capabilities again, because
-they may: differ on a per-physical device basis.
-====
```

Comment:
Was deletion of the `Note` intentional? Just asking as a sanity-check. I'm neutral about it, and am ok either way.

Reply:
There was a reason, which I can't remember device lost section?. But I also know I have been looking for this note for reference, so I am rethinking it.

Reply:
I asked for it above, because it feels kinda redundant. Otherwise similar note would need to be all over spec for consistency.